### ISSUE TITLE
docs: FAQ/doc on LB overview

### ISSUE THREAD

**Issue by mattklein123** [envoyproxy/envoy#5081]
The current LB arch overview guide for load balancing is too detailed and complicated for most new users to make sense of. We need an even high level "getting started with Envoy load balancing" that introduces high level concepts and then links to the more detailed arch documentation. For example, it might flow like:
1) What is load balancing
2) Distributed vs. centralized load balancing
3) Etc.

Basically, break https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/load_balancing into sub-pages with a high level overview first that helps people understand how to even start thinking about which options to choose.

**Comment by snowp** [comment]
I can take this. One question: when you say distributed vs centralized, are you then referring to active health checking being distributed and the EDS weights being centralized?

/assign @snowp

**Comment by mattklein123** [comment]
For distributed versus centralized I mean the general concept of doing distributed LB in which each LB operates independently (like in general what Envoy does) vs. having a global load balancer take part in the process (like what more advanced control planes do. A concrete example of this is doing zone aware load balancing either directly in Envoy (option 1 in the docs) vs. in the control plane (option 2 in the docs). Also, of course, most advanced people end up doing a combination of distributed and centralized. I think a general treatment of this concept would be very helpful as the existing docs are way too large and intimidating for the new user.

**Comment by mattklein123** [comment]
Also, thank you very much @snowp for offering to do this!!!

**Comment by mattklein123** [comment]
Calling this done with the work @snowp has done.
